mmdet3d/apis/test.py

```diff
@@ -35,12 +37,10 @@ def single_gpu_test(model,
             result = model(return_loss=False, rescale=True, **data)
 
         if show:
-            # Visualize the results of MMdetection3D model
-            # 'show_results' is MMdetection3D visualization API
-            if hasattr(model.module, 'show_results'):
+            # Visualize the results of MMDetection3D model
+            if isinstance(model.module, (Base3DDetector, Base3DSegmentor)):
                 model.module.show_results(data, result, out_dir)
-            # Visualize the results of MMdetection model
-            # 'show_result' is MMdetection visualization API
```

Comment:
this line should not be deleted, it should indicate `show_result` is MMDetection API?

Reply:
In the current code, I judge if `model` is a `Detector3D/Segmentor3D`. If yes, I will call `show_results`. If not, we assume `model` is a `Detector2D`, and we will perform 2D image visualization. I think this has fixed issue #419?